shared/actions/json/wallets.json

```diff
@@ -61,6 +74,37 @@
       "accountID": "Types.AccountID",
       "show?": "boolean",
     },
+    "sendPayment": {
+      "_description": "Perform sending a payment.",
+    },
+    "setBuildingAmount": {
+      "_description": "Set building amount",
+      "amount": "string",
+    },
+    "setBuildingCurrency": {
+      "_description": "Set building currency",
+      "currency": "string",
+    },
+    "setBuildingFrom": {
+      "_description": "Set building from",
+      "from": "string",
+    },
+    "setBuildingPublicMemo": {
+      "_description": "Set building public memo",
+      "publicMemo": "string",
```

Comment:
Should be hidden. Not strictly necessary but I think we should mask as much as possible